EmotiBitOscilloscope/src/ofApp.cpp

```diff
@@ -897,7 +900,25 @@ void ofApp::sendDataSelection(ofAbstractParameter& output) {
 			}
 		}
 	}
-
+	bool isSending = false;
+	// check if we are sending data out on any channel
+	for (int i = 0; i < sendDataList.size(); i++)
```

Comment:
Maybe, instead of performing another check (and looping through the the options, use the for-loop above to add/remove `None`?